### PR TITLE
Spanish translation

### PR DESCRIPTION
**Pull request by Allen1351**

### CODE REVIEW

**Review by sibir-ine — COMMENTED** [review]
@Allen1351 I've made comments for some errors I caught. Some can be committed from here, but a few will need manual corrections.

Also, loot.mo is unnecessary & should be deleted (LOOT builds it from loot.po already). You can prevent Poedit from generating one by going to `File` -> `Preferences...` -> `General` -> `Editing` -> `Automatically compile MO file when saving` & deselect it.

You should also go to `Catalog` -> `Properties` -> `Translation properties` -> `Project name and version:` & change that to `LOOT v0.16.1` since that's the current version of LOOT.

Finally, if you want, you can also go to `File` -> `Preferences...` -> `General` -> `Information about the translator` & enter a name/alias &/or an email that you'd like to be contacted by in case we or future translators have trouble reaching out to you. It's not necessary, but we'd appreciate it.

File: resources/l10n/es/LC_MESSAGES/loot.po [hunk 1]
```diff
@@ -1,945 +1,1269 @@
...
-msgid "Remove Game"
+#: src/gui/html/js/translateStaticText.ts:793
+msgid ""
+"Load order optimisation for Morrowind, Oblivion, Nehrim, Skyrim, Enderal, "
+"Skyrim Special Edition, Enderal Special Edition, Skyrim VR, Fallout 3, "
+"Fallout: New Vegas, Fallout 4 and Fallout 4 VR."
 msgstr ""
+"Optimización de la lista de carga para Morrowind, Oblivion, Nehrim, Skyrim, "
+"Enderal, Skyrim Special Edition, Enderal Special Edition, Skyrim VR, Fallout "
+"3, Fallout: New Vegas, Fallout 4 and Fallout 4 VR."
```
Comment (sibir-ine) [inline resources/l10n/es/LC_MESSAGES/loot.po:1005]: This "and" can be translated.

File: resources/l10n/es/LC_MESSAGES/loot.po [hunk 2]
```diff
@@ -1,945 +1,1269 @@
...
+msgstr "Información general"
 
-#: gui/main.cpp:961
+#: src/gui/html/js/translateStaticText.ts:597
 #, fuzzy
-msgid "BOSS: Warning"
-msgstr "BOSS: Trabajando..."
+#| msgid "Masterlist revision"
+msgid "Masterlist Revision"
+msgstr "Revisión de la MasterList"
```
Comment (sibir-ine) [inline resources/l10n/es/LC_MESSAGES/loot.po:787]:
```suggestion
msgstr "Revisión de la Masterlist"
```

File: resources/l10n/es/LC_MESSAGES/loot.po [hunk 3]
```diff
@@ -1,945 +1,1269 @@
...
 
-#: gui/main.cpp:971
-msgid "Error: Failed to get load order. %1%"
-msgstr ""
+#: src/gui/html/js/translateStaticText.ts:598
+msgid "Masterlist Date"
+msgstr "Fecha de la MasterList"
```
Comment (sibir-ine) [inline resources/l10n/es/LC_MESSAGES/loot.po:791]:
```suggestion
msgstr "Fecha de la Masterlist"
```

**Review by sibir-ine — COMMENTED** [review]
@Allen1351 I looked over the translation again & found a few more small issues, mostly with spaces & capitalization.

**Review by Ortham — CHANGES_REQUESTED** [review]
I've gone through the translation and commited a couple of formatting fixes. sibir's found some that I missed though, so please resolve them too.

**Pull request merged**